native~/Runtime/src/CameraManager.cpp

```diff
@@ -71,13 +73,19 @@ ViewState unityCameraToViewState(
   double horizontalFOV =
       2 * glm::atan(camera.aspect() * glm::tan(verticalFOV * 0.5));
 
+  const CesiumGeospatial::Ellipsoid& ellipsoid =
+      georeference != nullptr
```

Comment:
The nullptr check isn't needed here. `georeference` is a const reference, not a pointer.

Reply:
Unity objects, including CesiumGeoreference, actually overload `!= nullptr` to work as an existence check. You can see it elsewhere in the file!

Reply:
Ah!... not sure how I feel about that... but thanks for the explanation.

Reply:
The way to think about it is that instances of the generated classes in the DotNet namespace act like C# references. And just like it makes sense to compare a C# reference to null, it makes sense to compare an instance of these wrapper classes to nullptr. It makes no sense to have a stack-allocated instance of a C# object because C# classes are - by definition - heap allocated. So an instance of these classes really can't be anything other than a reference.

Reply:
It makes practical sense for sure. I think I was just caught a little off-guard since I was reviewing a .cpp file, and had my C++ glasses on.